CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.44.0 / 2020-12-02
+
+* [FEATURE] Add Slack, WeChat, Email, Pushover, and VictorOps receiver types to AlertmanagerConfig CRD. #3618 #3619 #3692 #3697 #3701
+* [FEATURE] Add option to allow configuring object storage for Thanos. #3668
+* [FEATURE] Add TLS support for remote read. #3714
+* [ENHANCEMENT] Include EnforcedSampleLimit as a metric. #3617
+* [ENHANCEMENT] Adjust config reloader memory requests and limits. #3660
+* [ENHANCEMENT] Add settings for alertmanager clusters #3663
+* [ENHANCEMENT] Handle all possible reciever types in AlertmanagerConfig. #3689
+* [BUGFIX] Add sanity check for empty Probe targets. #3637
+* [BUGFIX] Fix usage of --*-default-base-image flags. #3642
+* [BUGFIX] Fix matching labels with empty values when using Exists/NotExists operators. #3686
+* [BUGFIX] Fix child routes support in AlertmanagerConfig. #3703
```

Comment:
We wanted this to have a CHANGE mention https://github.com/prometheus-operator/prometheus-operator/pull/3703#issuecomment-733807745

Reply:
Thanks! This slipped my attention. Fixed.